src/main/java/cs2103/v15_1j/jimjim/command/ChangeCommand.java

```diff
@@ -122,6 +123,14 @@ public UIFeedback execute(ControllerStates conStates) {
             	}
             }
             
+            if (actual instanceof Event) {
+            	if (tempEvent.getEndDateTime().isBefore(tempEvent.getStartDateTime())) {
```

Comment:
It's better to avoid naming a variable temp because they are hard to understand without context. Just call it event. There is only 1 event being talked about here, why add the word temp?